packages/flutter_tools/test/commands.shard/hermetic/config_test.dart

```diff
@@ -67,7 +57,7 @@ void main() {
         '${allFeatures.where((Feature e) => e.configSetting != null).map((Feature e) => '  ${e.configSetting}: (Not set)').join('\n')}'
         '\n\n',
       );
-    }, overrides: <Type, Generator>{Usage: () => testUsage});
```

Comment:
Not sure if this override was ever necessary since we don't make assertions against `testUsage` in this test (`testUsingContext` already inserts a null/no-op implementation for `Usage` and `Analytics`).